docs/guides/configure.md

```diff
@@ -1,6 +1,8 @@
 # Configure
 
-**Concrete** can be customized using `Configuration`s:
+This document provides instructions on how to customize **Concrete** using Configurations in Python and describes various configuration options available.
```

Comment:
That's a bit weird to say customize `Concrete` is more about customize the `compilation pipeline`

Reply:
Ok I updated it, thank you